audioread/ffdec.py

```diff
@@ -264,12 +264,14 @@ def close(self):
             # ffmpeg closes normally on its own, but never updates
             # `returncode`.
             self.proc.poll()
+            self.proc.stdout.close()
+            self.proc.stderr.close()
 
             # Kill the process if it is still running.
             if self.proc.returncode is None:
                 self.proc.kill()
                 self.proc.wait()
-                self.devnull.close()
+        self.devnull.close()
```

Comment:
Again, since this is separate from the above handling of the subprocess, it would be a good idea to put a blank line above this one. I'm not sure if an explanatory comment would be useful to say something about why this is necessary.

Reply:
I added a comment anyway -- feel free to remove it if you believe it is too verbose. :)